### ISSUE TITLE
CLI creates two Node Clusters from a single `create` command

### ISSUE THREAD

**Issue by joegoggins** [tutumcloud/cli#94]
When I call `tutum nodecluster create` in a programmatic context from within a bash script, triggered from a configuration management system without a tty, it results in provisioning 2 clusters instead of 1. This was maddeningly difficult to tease out, but I've taken good notes on the behavior that are worth sharing in hopes that this can be replicated and fixed in the [dockercloud-cli project](https://github.com/docker/dockercloud-cli). 

### Observed behavior

When the script executes:

    tutum nodecluster create --sync --tag t1 --target-num-nodes 1 --disk 20 --aws-vpc-id vpc-xxxxxxxx --aws-vpc-subnet subnet-xxxxxxxx --aws-vpc-security-group sg-xxxxxxxx t1 aws us-east-1 t2.small

A few minutes later I get the following on stdout (nothing on stderr)

       In progress..........................
       Success
       16306239-ccb8-4782-a3ab-16935464ab70

**But the command doesn't exit after that, instead it continues to block for another few minutes, while in background it's creating a second NodeCluster with a different UUID, different from the one displayed.** You can see it begin to provision the second node cluster in the Tutum UI immediately after it outputs the above.

### Work Around 

The work around I'm gonna go with for now is to simply not try to use this automation script for auto-creating nodeclusters, that is frustrating and blocks some ambitious test driven infrastructure goals we're working toward.

Please let me know if I can clarify, or how I can help get this resolved.

Thanks!

-joe

**Comment by tifayuki** [comment]
Hi @joegoggins 

Thank you for reporting the issue.
I tested myself with the `nodecluster create` command and it works fine on my side. Can you reproduce this issue?

If so, can you run the command with `tutum --debug nodecluster create ...` and send us the logs.( with`'Authorization': 'Basic xxxxxxxx'` removed in the log)

It could be helpful.

Thank you

**Comment by joegoggins** [comment]
Thanks so much for the quick reply--I'll get you some detailed logs in the next day or two!

**Comment by joegoggins** [comment]
Hey there @tifayuki , I generated detailed logs, scrubbed the basic auth info and emailed them to you (didn't feel like sharing them all publicly). To me they appear to indicate either a cloud-side bug or perhaps the cli bug that fails to log what it's actually doing. Let me know if I can provide more data/info. Thanks.

**Comment by tifayuki** [comment]
Hi @joegoggins, there is nothing weird in your log.

From the source code (https://github.com/tutumcloud/cli/blob/staging/tutumcli/commands.py#L1294) the command does exit after printing the uuid of the ndoe cluster. 

Could it be the problem of the ruby console? Does it happen every time or occasionally? Does it happen when you run the command manually in shell?

**Comment by joegoggins** [comment]
> Could it be the problem of the ruby console?

I don't think so. It happens when calling it from a bash script, whether directly on the command line or in a ruby console.

> Does it happen every time or occasionally?

It happens every time.

> Does it happen when you run the command manually in shell?

No, it works if executed directly.
It does not work when called from within a `#/bin/bash` script

One thing I had not mentioned in case it's useful. Seeing this behavior on Ubuntu 14.04.
